packages/orbit-components/src/Separator/types.d.ts

```diff
@@ -5,98 +5,88 @@ import type * as Common from "../common/types";
-  | "paletteRedNormalHover"
-  | "paletteRedNormalActive"
-  | "paletteRedDark"
-  | "paletteRedDarkHover"
-  | "paletteRedDarkActive"
-  | "paletteRedDarker"
-  | "paletteSocialFacebook"
-  | "paletteSocialFacebookHover"
-  | "paletteSocialFacebookActive"
-  | "paletteWhite"
-  | "paletteWhiteNormal"
-  | "paletteWhiteHover"
-  | "paletteWhiteActive";
+export type BorderColorClass =
+  | "border-blue-light"
+  | "border-blue-light-hover"
+  | "border-blue-light-active"
+  | "border-blue-normal"
+  | "border-blue-normal-hover"
+  | "border-blue-normal-active"
+  | "border-blue-dark"
+  | "border-blue-dark-hover"
+  | "border-blue-dark-active"
+  | "border-blue-darker"
+  | "border-bundle-basic"
+  | "border-bundle-medium"
+  | "border-cloud-light"
+  | "border-cloud-light-hover"
+  | "border-cloud-light-active"
+  | "border-cloud-normal"
+  | "border-cloud-normal-hover"
+  | "border-cloud-normal-active"
+  | "border-cloud-dark"
+  | "border-cloud-dark-hover"
+  | "border-cloud-dark-active"
+  | "border-green-light"
+  | "border-green-light-hover"
+  | "border-green-light-active"
+  | "border-green-normal"
+  | "border-green-normal-hover"
+  | "border-green-normal-active"
+  | "border-green-dark"
+  | "border-green-dark-hover"
+  | "border-green-dark-active"
+  | "border-green-darker"
+  | "border-ink-dark"
+  | "border-ink-dark-hover"
+  | "border-ink-dark-active"
+  | "border-ink-light"
+  | "border-ink-light-hover"
+  | "border-ink-light-active"
+  | "border-ink-normal"
+  | "border-ink-normal-hover"
+  | "border-ink-normal-active"
+  | "border-orange-light"
+  | "border-orange-light-hover"
+  | "border-orange-light-active"
+  | "border-orange-normal"
+  | "border-orange-normal-hover"
+  | "border-orange-normal-active"
+  | "border-orange-dark"
+  | "border-orange-dark-hover"
+  | "border-orange-dark-active"
+  | "border-orange-darker"
+  | "border-product-light"
+  | "border-product-light-hover"
+  | "border-product-light-active"
+  | "border-product-normal"
+  | "border-product-normal-hover"
+  | "border-product-normal-active"
+  | "border-product-dark"
+  | "border-product-dark-hover"
+  | "border-product-dark-active"
+  | "border-product-darker"
+  | "border-red-light"
+  | "border-red-light-hover"
+  | "border-red-light-active"
+  | "border-red-normal"
+  | "border-red-normal-hover"
+  | "border-red-normal-active"
+  | "border-red-dark"
+  | "border-red-dark-hover"
+  | "border-red-dark-active"
+  | "border-red-darker"
+  | "border-social-facebook"
+  | "border-social-facebook-hover"
+  | "border-social-facebook-active"
+  | "border-white"
+  | "border-white-normal"
+  | "border-white-hover"
+  | "border-white-active";
 
-export interface CommonProps extends Common.SpaceAfter {
-  indent?: Indent;
-  align?: Align;
+export interface Props extends Common.SpaceAfter {
+  readonly indent?: Indent;
+  readonly align?: Align;
+  readonly borderColor?: BorderColorClass;
+  readonly type?: "solid" | "dashed" | "dotted" | "double" | "none";
```

Comment:
This is a good simplification and a good change. However, there is one catch:
In the past, if someone had explicitly `type="none"` without any `color` defined, the separator would be visible. But now it is not.

I am not against the change. But we need this change to be clear and documented, besides the others that you mention on the PR description.